README_ADMIN.md

```diff
@@ -0,0 +1,218 @@
+$ rsconnect-admin content search --unpublished
+
+# published content where the python version is at least 3.9.0
+$ rsconnect-admin content search --published --py-version ">=3.9.0"
+
+# published content where the R version is exactly 3.6.3
+$ rsconnect-admin content search --published --r-version "==3.6.3"
+
+# published content where the content type is a static RMD
+$ rsconnect-admin content search --content-type rmd-static
+
+# published content where the content type is either shiny OR fast-api
+$ rsconnect-admin content search --content-type shiny --content-type python-fastapi
+
+# any content, published or unpublished, where the title contains the text "Stock Report"
+$ rsconnect-admin content search --title-contains "Stock Report"
+
+# published content, results are ordered by when the content was last deployed
+$ rsconnect-admin content search --published --order-by last_deployed
+
+# published content, results are ordered by when the content was created
+$ rsconnect-admin content search --published --order-by created
+```
+
+
+### Building Content
+
+RStudio Connect caches R and Python packages in the configured
+[`Server.DataDir`](https://docs.rstudio.com/connect/admin/appendix/configuration/#Server.DataDir).
+Sometimes, these package caches become stale and need to be rebuilt. This refresh
+occurs automatically when an RStudio Connect user visits the content via the
+RStudio Connect dashboard. Some content, however, is high priority or not accessed
+frequently via the dashboard (API content, emailed reports). In these cases, administrators
+can preemptively build specific content items using the `rsconnect-admin build` subcommands.
+This way the user does not have to pay the build cost when the content is accessed next.
+
+The following are some common scenarios where performing a content build might be necessary:
+
+- OS upgrade
+- changes to gcc or libc libraries
+- changes to Python or R installations
+- switching from source to binary package repositories or vice versa
+- migrating from local content execution to remote content execution with Kubernetes
+
+Administrators should use the [`rsconnect-admin content search`](#search) subcommand to help
+identify content items for building.
+
+To build a specific content item, first `add` it to the list of content that is
+"tracked" for building using its GUID.
+
+```bash
+# add a single content item by guid
+$ rsconnect-admin build add --guid 4ffc819c-065c-420c-88eb-332db1133317
+```
+
+Administrators can view currently "tracked" content items using the
+`rsconnect-admin build ls` subcommand.
+
+```bash
+# list all tracked content
+$ rsconnect-admin build ls
+
+# list tracked content that still needs to be built
+$ rsconnect-admin build ls --status NEEDS_BUILD
+
+# list tracked content that has already been built successfully
+$ rsconnect-admin build ls --status COMPLETE
+
+# list tracked content that failed to build
+$ rsconnect-admin build ls --status=ERROR
```

Comment:
This one uses `status=ERROR`, but the one above used `status COMPLETE`. Which is correct since it seems like these should be consistent. (Aborted below is the same way)